src/Symfony/Component/Notifier/Bridge/Discord/DiscordTransport.php

```diff
@@ -29,6 +29,8 @@ final class DiscordTransport extends AbstractTransport
 {
     protected const HOST = 'discord.com';
 
+    private const MAX_SUBJECT_LENGTH = 2000;
```

Comment:
it's called TEXT_LIMIT in the Slack bridge 5.2
maybe SUBJECT_LIMIT here? or another name in the Slack bridge?